homeassistant/components/yeelight/light.py

```diff
@@ -502,7 +502,7 @@ def set_effect(self, effect) -> None:
 
     def turn_on(self, **kwargs) -> None:
         """Turn the bulb on."""
-        import yeelight
+        from yeelight import BulbException
```

Comment:
Could you move the imports to the top of the file, now that this is possible?

Reply:
Is it really possible now ? Some time ago I've asked and response was there is still not ready ( after manifests were added ). If yes if it officially supported. It would be great and simplify code a lot.